distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -133,19 +133,16 @@ type Table
        - columns: Column selection criteria or vector of column names.
        - reorder: By default, or if set to `False`, columns in the output will
          be in the same order as in the input table. If `True`, the order in the
-         output table will match the order in the columns list.
+         output table will match the order in the columns list. If a column is
+         matched by multiple selectors in reorder mode, it will be placed at
+         the position of the first one matched.
        - on_problems: Specifies how to handle problems if they occur, reporting
          them as warnings by default.
 
          The following problems can occur:
          - If a column in columns is not in the input table, a
            `Missing_Input_Columns`.
-         - If duplicate columns, names or indices are provided, a
-           `Duplicate_Column_Selectors`.
          - If a column index is out of range, a `Column_Indexes_Out_Of_Range`.
-         - If two distinct indices would refer to the same column, a
-           `Input_Indices_Already_Matched`, indicating that the additional
-           indices will not introduce additional columns.
          - If there are no columns in the output table, a `No_Output_Columns` is
            raised as an error regardless of the problem behavior, because it is
            not possible to create a table without any columns.
```

Comment:
As these are now errors should we have them as `!` on the type signature.

Reply:
I'll amend this in the next pass along with the doc update.